src/Database/Redis/Cluster.hs

```diff
@@ -0,0 +1,128 @@
+{-# LANGUAGE DeriveDataTypeable #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE ViewPatterns #-}
+module Database.Redis.Cluster
```

Comment:
Since you haven't mentioned this in PRs description (I think), I want to remind you to also not forget putting the Haddock everywhere in the module.